blob/service_test.go

```diff
@@ -328,6 +330,24 @@ func TestBlobService_Get(t *testing.T) {
 				require.NoError(t, proof.equal(*newProof))
 			},
 		},
+		{
+			name: "internal error",
```

Comment:
what about testing the case where there's a partial internal error?

e.g. some blobs returned for other namespaces, but an internal error for one of the namespaces?